sleap_io/model/skeleton.py

```
@@ -229,103 +260,330 @@
+        dst: NodeOrIndex | None = None,
+    ):
         """Add an `Edge` to the skeleton.
 
         Args:
-            src: The source `Node` or name of the source node.
-            dst: The destination `Node` or name of the destination node.
+            src: The source node specified as a `Node`, name or index.
+            dst: The destination node specified as a `Node`, name or index.
         """
+        edge = None
+        if type(src) == tuple:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)